src/status_im/subs/wallet/wallet_test.cljs

```diff
@@ -12,6 +12,14 @@
 (use-fixtures :each
               {:before #(reset! rf-db/app-db {})})
 
+(def ^:private accounts-with-tokens
```

Comment:
A quick side note @mmilad75. In theory, we shouldn't worry about encapsulation (private/public vars) in namespaces that won't ever be required elsewhere (like test namespaces) because there's no API to be exposed. The same argument works for preview namespaces. In these namespaces, everything can be public. Consider that nobody needs to refactor a component preview namespace or a test namespace and take in consideration vars exposed from them.

cc'ing @ulisesmac @briansztamfater just to give some visibility :)